config.toml

```diff
@@ -144,7 +144,7 @@ fullversion = "v1.19.4"
 version = "v1.19"
 githubbranch = "v1.19.4"
 docsbranch = "release-1.19"
-deprecated = false
+deprecated = true
```

Comment:
👍  thank you for catching this! I will add this to the docs role handbook for the future leads

Reply:
👍 👍